src/ci/run.sh

```diff
@@ -25,6 +25,8 @@ source "$ci_dir/shared.sh"
 
 if [ "$TRAVIS" == "true" ] && [ "$TRAVIS_BRANCH" != "auto" ]; then
```

Comment:
I believe `TRAVIS` is `false` on AppVeyor. We should generalize this condition to cover AppVeyor as well.

Reply:
Indeed it is! This is only here, though, for making PR builds a little less noisy. We only do full bors builds on AppVeyor so I think I'll leave this for now until that situation changes